src/graphql/blocks.ts

```diff
@@ -65,6 +66,33 @@ async function tsToBlockNum(network, ts) {
   return to.number;
 }
 
+async function tsToBlockNum(network, ts) {
```

Comment:
I feel the name of the functions `calculateBlockNum` and `tsToBlockNum` are confusing, they both sound the same, I think it would better to do 3 functions `tsToBlockWithApi`, `tsToBlockWithNode` and `tsToBlock` (which call the 2 functions.

Reply:
Sounds good :ok_hand: